mp/src/game/client/momentum/ui/HUD/hud_stickybombcharge.cpp

```diff
@@ -57,9 +68,35 @@ CHudStickyCharge::CHudStickyCharge(const char *pElementName)
     LoadControlSettings("resource/ui/HudStickyCharge.res");
 }
 
+void CHudStickyCharge::FireGameEvent(IGameEvent* pEvent)
+{
+    auto *pLocal = C_MomentumPlayer::GetLocalMomPlayer();
+    if (!pLocal)
+        return; 
+
+    if (FStrEq(pEvent->GetName(), "zone_enter") && pEvent->GetInt("num") == 1)
+    {
+        // Turn charge meter red while inside start zone to indicate that stickies can't be charged
+        m_pChargeLabel->SetVisible(false);
+        m_pChargeMeter->SetSubdivMarksVisible(false);
+        m_pChargeMeter->SetFgColor(m_cChargeDisabled);
+        m_pChargeMeter->SetProgress(1.0f);
+    }
+    else
```

Comment:
Careful, you may still want to check if the num is 1 here, as if it's `zone_enter` but on stage 2, this else will be called. You can probably make it early return if the num isn't 1.

Reply:
oops!